app/controllers/organizations/adopter_fosterer/adopted_pets_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Organizations
+  module AdopterFosterer
+    class AdoptedPetsController < Organizations::BaseController
+      before_action :context_authorize!
+      layout "adopter_foster_dashboard"
+
+      def index
```

Comment:
Could we instead query the `Match` model to find all matches with this `adopter_foster_account_id` where the `match_type: :adoption`?